ha/util/message_bus.py

```diff
@@ -145,6 +148,14 @@ def stop(self):
 class MessageBus:
     ADMIN_ID = "ha_admin"
 
+    @staticmethod
+    def init():
```

Comment:
Please add explanation here as to why it is not called inside the get_consumer / get_producer methods

Reply:
If we do MessageBus.init() in every get_consumer & get producer call & if our service uses both consumer & producer then MessageBus is initialized twice which will cause an extra overload to MessageBus. Also in future if utils have added check that MessageBus must be initialized only once per service then our message_bus wrapper will fail.

Reply:
Added comment.